html-forms/index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>HTML Forms</title>
+</head>
+<style>
+  body {
+    background-image: linear-gradient(to right, rgb(182, 31, 182) , rgba(0, 0, 255, 0.836));
+  }
+  div {
+    font-weight: 600;
+  }
+</style>
+<body>
+  <h1>Coding Form!</h1>
+  <form action="">
+    <div>
+      <div>Email</div>
+      <input type="email" id="email">
+    </div>
+
+    <div>
+      <div>Why is coding cool?</div>
+      <select name="select" id="select">
+        <option value="Select an option">Select an option</option>
+        <option value="Because it's interesting">Because it's interesting</option>
+        <option value="Because it's fun">Because it's fun</option>
+        <option value="Because it's powerful">Because it's powerful</option>
+      </select>
+
+      <div>
+        <div>Favorite Language (choose one)</div>
+        <input type="radio" id="html" name="language" value="html"/> HTML
+        <input type="radio" id="css" name="language" value="css"/> CSS
+        <input type="radio" id="javascript" name="language" value="javascript"/> JavaScript
+      </div>
+
+      <div>
+        <div>Would you like to sign up for more information?</div>
+        <input type="checkbox"/> Sign me up!
+      </div>
+
+      <div>
+        <div>Comments</div>
```

Comment:
This should be a `label` element. Make sure you use a `for` attribute with a value that matches the `id` of the form control element this is labelling!